source/client/app/Minecraft.cpp

```diff
@@ -508,11 +508,12 @@ void Minecraft::tickInput()
 			}
 			else if (getOptions()->isKey(KM_DROP, keyCode))
 			{
-				int itemID = m_pLocalPlayer->m_pInventory->getSelectedItemId();
-				if (itemID > 0)
+				ItemInstance *item = m_pLocalPlayer->m_pInventory->getSelected();
```

Comment:
I called it getSelected because that's what Mojang did. I can rename it if you want, but I feel like it's pretty self-explanatory. Also, as of 0.9.2, it looks like getSelectedItemId didn't exist, only getSelected.